tests/test_check.py

```diff
@@ -77,6 +77,32 @@ def test_check_passing_distribution(monkeypatch):
     assert renderer.render.calls == [pretend.call("blah", stream=warning_stream)]
 
 
+def test_check_passing_distribution_with_none_renderer(monkeypatch):
```

Comment:
I think I see what you're getting at here, but I think there's a way to write it with less patching, and more coverage of the desired behavior.

Noting that two content types already return `None`:

https://github.com/pypa/twine/blob/895ac8af383b99d9b1673f2d7edcea92e4b5a429/twine/commands/check.py#L31-L36

How about we parameterize the test with those? ([source](https://github.com/bhrutledge/twine/blob/0bdddf3f60f4363f52b551888cb1f7ca9544fedf/tests/test_check.py#L81-L101))

```diff
@@ -16,2 +16,3 @@ import io
 import pretend
+import pytest
 
@@ -79,6 +80,6 @@ def test_check_passing_distribution(monkeypatch):
 
-def test_check_passing_distribution_with_none_renderer(monkeypatch):
+@pytest.mark.parametrize("content_type", ["text/plain", "text/markdown"])
+def test_check_passing_distribution_with_none_renderer(content_type, monkeypatch):
     """Test when a distribution passes twine check with renderers returning None"""
 
-    renderer = pretend.stub(render=pretend.call_recorder(lambda *a, **kw: None))
     package = pretend.stub(
@@ -86,3 +87,3 @@ def test_check_passing_distribution_with_none_renderer(monkeypatch):
             "description": "blah",
-            "description_content_type": "text/markdown",
+            "description_content_type": content_type,
         }
@@ -91,3 +92,2 @@ def test_check_passing_distribution_with_none_renderer(monkeypatch):
 
-    monkeypatch.setattr(check, "_RENDERERS", {None: None})
     monkeypatch.setattr(commands, "_find_dists", lambda a: ["dist/dist.tar.gz"])
@@ -102,4 +102,2 @@ def test_check_passing_distribution_with_none_renderer(monkeypatch):
 
-    assert renderer.render.calls == []
-
```

Reply:
Parametrized the tests in https://github.com/pypa/twine/pull/621/commits/32013d06dace2803694ff04a0a81333ec68bc9d6. I am also guilty of not looking at the code more closely to write a more efficient test